SUMMARY.md

```diff
@@ -0,0 +1,19 @@
+## Table of Contents
+
+* [Ternary Operator](content/TernaryOperator/TernaryOperator.q.md) ** - minutes**
+    > In this lesson, you will learn how to simplify the condition of if/else statements using ternary operator. You will also learn the advantages of using ternary operator as an expression.
+
+* [Type Hinting](content/TypeHinting/TypeHinting.q.md) ** - minutes**
+    > Discover how you can use type hinting to identify the expected data type being passed as arguments in a function declaration. 
+ 
+* [Sessions](content/Sessions/Sessions.q.md) ** - minutes**
+    > Learn how to start a PHP session, get PHP session variable values, modify a PHP session variable, and destroy a PHP session.
+
+* [Cookies](content/Cookies/Cookies.q.md) ** - minutes**
+    > Learn how to create/retrieve cookies, modify cookie values, delete cookies, and check if cookies are enabled.
+
+* [Exceptions](content/Exceptions/Exceptions.q.md) ** - minutes**
```

Comment:
Please try to read further about the different Exception Classes. I think it is not advisable to include custom exceptions here since `Exception Classes` for me is a very complicated topic that I think we can break into two: Basic Exception classes that focuses only on the hierarchy and catching; Second will be the Throwing of Exceptions using `throws` . Please try to check. Thanks.